tasks/main.yml

```diff
@@ -25,7 +25,10 @@
   args:
     creates: /etc/php5/cli/conf.d/20-mcrypt.ini
 
-- name: Copy php-fpm configuration
+- name: Copy php-fpm php.ini configuration
+  template: src=php.ini dest=/etc/php5/fpm/
```

Comment:
Whoops, I'm not entirely accurate there. Try something like

```
php_admin_value[open_basedir] = /var/www/
php_admin_value[upload_tmp_dir] = /var/www/tmp/
```

In wordpress.conf. FPM .ini seem to be a bit more finicky...